package.json

```diff
@@ -6,14 +6,15 @@
     "build": "next build",
     "db:push": "dotenv drizzle-kit push:mysql",
     "migrate": "dotenv drizzle-kit migrate",
-    "dev": "next dev --turbo",
```

Comment:
why is turbo gone?

Reply:
It's not working properly with the new font. Idk why